.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea
+.vscode
+
+# misc
+.DS_Store
+.env.local
```

Comment:
Are you using .env values? If so, can you please add a .env.example so we know what to look for?